synapse/lib/service.py

```diff
@@ -124,6 +124,14 @@ class SvcProxy(s_eventbus.EventBus):
     def __init__(self, sbus, timeout=None):
         s_eventbus.EventBus.__init__(self)
 
+        self.byiden = {}
```

Comment:
these had to be moved otherwise the service callbacks could fire prior to the service proxy having items to accessing

Reply:
good call!